indent/svelte.vim

```diff
@@ -86,7 +86,8 @@ endif
 " Settings {{{
 "
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-setlocal sw=2 ts=2
+let &l:shiftwidth=&g:shiftwidth
+let &l:tabstop=&g:tabstop
```

Comment:
```suggestion

```

Reply:
Hi, I agree with you to remove the setting of `shiftwidth` and `tabstop`, but I don't think we have to set it explicitly as Vim will automatically set `shiftwidth` value in the order from local to global, then `tabstop`.

Meanwhile, after your commit, I'll add code to keep its value across loading multiple sub-syntax indent files.

In then end, users can set it by 
```vim
autocmd FileType svelte :setlocal shiftwidth=2
```

Reply:
I did try removing it completely and while that worked with the vue plugin it caused some odd behavior while typing `function() {` in the svelte plugin. The curly brace gets pushed to the next line. Wasn't sure if that was a problem caused by other plugins I had installed though.

Reply:
I'm not sure. Does it only happen if removing `setlocal sw=2 ts=2`?
 It probably was caused by other plugins, but feel free to create a new issue for that so we can take a look at it. You can also try disabling other plugins temporarily to narrow the problem.